index.html

```diff
@@ -3386,6 +3484,150 @@ <h3><dfn>Writing content</dfn></h3>
     </section>
   </section>
 
+  <section id="making-content-read-only">
+    <h3><dfn>Making content read only</dfn></h3>
+    <p>
+      This section describes how to make an <a>NFC tag</a> read only when it is
+      in proximity range. At any time there is a maximum of one request for an
+      <a>origin</a> until the <a>NFC tag</a> is made read only or the operation
```

Comment:
I am wondering whether we should say read-only instead with a hyphen

Reply:
Fixed.